soc/arm/st_stm32/stm32f1/dts_fixup.h

```diff
@@ -209,6 +219,21 @@
 #define DT_TIM_STM32_8_CLOCK_BITS	DT_ST_STM32_TIMERS_40013400_CLOCK_BITS
 #define DT_TIM_STM32_8_CLOCK_BUS	DT_ST_STM32_TIMERS_40013400_CLOCK_BUS
 
+#define DT_CAN_1_BASE_ADDRESS		DT_ST_STM32_CAN_40006400_BASE_ADDRESS
+#define DT_CAN_1_BUS_SPEED		DT_ST_STM32_CAN_40006400_BUS_SPEED
+#define DT_CAN_1_NAME			DT_ST_STM32_CAN_40006400_LABEL
+#define DT_CAN_1_IRQ_TX			DT_ST_STM32_CAN_40006400_IRQ_TX
+#define DT_CAN_1_IRQ_RX0		DT_ST_STM32_CAN_40006400_IRQ_RX0
+#define DT_CAN_1_IRQ_RX1		DT_ST_STM32_CAN_40006400_IRQ_RX1
+#define DT_CAN_1_IRQ_SCE		DT_ST_STM32_CAN_40006400_IRQ_SCE
+#define DT_CAN_1_IRQ_PRIORITY		DT_ST_STM32_CAN_40006400_IRQ_0_PRIORITY
+#define DT_CAN_1_SJW			DT_ST_STM32_CAN_40006400_SJW
+#define DT_CAN_1_PROP_SEG		DT_ST_STM32_CAN_40006400_PROP_SEG
+#define DT_CAN_1_PHASE_SEG1		DT_ST_STM32_CAN_40006400_PHASE_SEG1
+#define DT_CAN_1_PHASE_SEG2		DT_ST_STM32_CAN_40006400_PHASE_SEG2
+#define DT_CAN_1_CLOCK_BUS		DT_ST_STM32_CAN_40006400_CLOCK_BUS
+#define DT_CAN_1_CLOCK_BITS		DT_ST_STM32_CAN_40006400_CLOCK_BITS
```

Comment:
Why do we define them again?

Reply:
Ah, my work was before c180e05, thanks for pointing that out